lib/functoria/install.mli

```diff
@@ -34,9 +34,12 @@ val pp : t Fmt.t
 val pp_opam : t Fmt.t
 (** Print the opam rules to install [t] *)
 
-val dune : context_name:string -> t -> Dune.t
+val dune : context_name:string * string -> t -> Dune.t
```

Comment:
`string * string` is a bit confusing, we don't know which one is the host and which one is the target context name